files/propertyInspector/commandToggle.js

```diff
@@ -0,0 +1,34 @@
+function connectElgatoStreamDeckSocket(inPort, inPropertyInspectorUUID, inRegisterEvent, inInfo, inActionInfo) {
+	websocket = new WebSocket("ws://localhost:" + inPort);
+	inInfo = JSON.parse(inInfo);
+	inActionInfo = JSON.parse(inActionInfo);
+
+	websocket.onopen = () => {
+		websocket.send(JSON.stringify({
+			event: inRegisterEvent,
+			uuid: inPropertyInspectorUUID
+		}));
+	};
+
+	let down = document.getElementById("S1Down");
```

Comment:
I would appreciate if you kept these element IDs lowercase, as is the convention. Also, if it's `down2` and `up2`, the first set should be `down1` and `up1`. Just nitpicking though.